src/strapi_kit/export/relation_resolver.py

```
@@ -170,3 +178,223 @@ def build_relation_payload(
+        Args:
+            component_data: Component data dictionary
+            component_uid: Component UID for schema lookup
+            schema_cache: Schema cache for component lookups
+            prefix: Field path prefix for nested fields
+
+        Returns:
+            Dictionary mapping prefixed field paths to lists of IDs
+        """
+        try:
+            component_schema = schema_cache.get_component_schema(component_uid)
+        except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)